explainer.md

```diff
@@ -111,34 +113,26 @@ code](demos/polyfill.js), which roughly implements this logic.
 ## LaunchEvent
 
     interface LaunchEvent : ExtendableEvent {
-      readonly attribute USVString url;
+      readonly attribute Request request;
       readonly attribute DOMString? clientId;
-
-      void handleLaunch(Promise<any> promise);
     }
 
-* `handleLaunch` is analogous to
+* `waitUntil` delays the user agent from launching and waits for the promise.
+  If the promise rejects, the default launch behaviour resumes.
```

Comment:
I'm not sure about treating failure as default. (What about success? Does this basically just mean we ignore the result of the promises?)

I think we want to either:
- Ignore the success/failure of the promises passed to `waitUntil`, or
- If any promise passed to `waitUntil` rejects (remember there can be multiple of them), stop the launch, as if `preventDefault` was called.